api/main.py

```diff
@@ -65,7 +66,13 @@ def human(id):
     r = requests.get(f"https://aisafety.info/questions/{id}")
     log(f"clicked followup '{json.loads(r.text)['data']['title']}': https://stampy.ai/?state={id}")
 
-    return Response(r.text, mimetype='application/json')
+    # run a regex to replace all relative links with absolute links. Just doing
+    # a regex for now since we really don't need to parse everything out then
+    # re-serialize it for something this simple.
+    # <a href=\"/?state=6207\"> -> <a href=\"https://stampy.ai/?state=6207\">
+    text = re.sub(r'<a href=\\"/\?state=(\d+)\\">', r'<a href=\"https://stampy.ai/?state=\1\">', r.text)
```

Comment:
Probably better to use "https://aisafety.info/?state=" instead of stampy.ai

Reply:
resolved.